includes/Customer.php

```diff
@@ -55,7 +55,7 @@ class Customer {
 	/**
 	 * Collect customer data
 	 *
-	 * @return array of customer data
+	 * @return array of customer data or false if none
```

Comment:
@circlecube We should probably coerce `false` to be an empty array and keep a consistent return type.

Reply:
Does this look better, @wpscholar ?